### DISCUSSION TITLE
snaplet.dev copy supabase instance error

### DISCUSSION THREAD

**Discussion by marc-on-github** [supabase/supabase#5624] [Questions]
In the last newsletter, you recommended snaplet.dev. I was looking to create a dev environment based on the prod, so this solution came at the perfect time.
However, I have the problem that after copying from the prod environment to the dev environment, the dev environment is not available when writing data via the @supabase/supabase-js and the private key. I get this error:
"message": "permission denied for schema public",
"code": "42501",
"details": null,
"hint": null

Does anyone have any idea what I am doing wrong, or what data I need to exclude so that this error does not occur?
Thanks!

**Comment by peterp** [comment]
Hey @marc-on-github,

Peter from Snaplet here! Snapet automatically transforms your `auth.Users` table by creating a configuration that removes some of the private information:
```json
{
  "auth": {
      "users": {
        "email": "${internet.email()}",
        "phone": "${phone.phoneNumber(\"###-####\")}"
      },
      "refresh_tokens": {
        "token": "${random.uuid()}"
      }
    }
}
```

You can remove these, by editing your snapshot configuration in `Database -> Data Editor`, and removing those transformations.
